index-scheduler/src/snapshots/index_scheduler__tests__import_vectors-15.snap

```diff
@@ -6,10 +6,6 @@ expression: doc
   "doggo": "Intel",
   "breed": "beagle",
   "_vectors": {
-    "A_fakerest": {
```

Comment:
I would expect the vector here to still be present?

maybe let's use the new boolean to force displaying the vectors?

Reply:
Here, we can't because we're not in meilisearch but in the index-scheduler directly.
We need to look into arroy to find the vector back, but since we were ignoring its value anyway, I don’t think it’s an issue if I don't try to retrieve it, right?

Reply:
hmm, we're ignoring its value, but that there is still a user-provided vector is important I guess. Now I need to read the test again to check the context

Reply:
Yes, we do that!
```rust
            // for consistency with the below
            #[allow(clippy::get_first)]
            let IndexEmbeddingConfig { name, config: fakerest_config, user_defined } =
                configs.get(0).unwrap();
            insta::assert_snapshot!(name, @"A_fakerest");
            insta::assert_debug_snapshot!(user_defined, @"RoaringBitmap<[]>");
            insta::assert_json_snapshot!(fakerest_config.embedder_options);
```

Reply:
OK then